docs/src/constructors.md

```diff
@@ -27,8 +36,9 @@ refer to as parent objects. They behave a lot like Julia types.
 
 Consider the following simple example, to create a Flint multiprecision integer:
 
-```
-n = ZZ("12345678765456787654567890987654567898765678909876567890")
+```jldoctest
+julia> n = ZZ("12345678765456787654567890987654567898765678909876567890")
```

Comment:
I wonder why we use a sting literal here, instead of an integer literal, i.e.,
```suggestion
julia> n = ZZ(12345678765456787654567890987654567898765678909876567890)
```

Reply:
Maybe in the old (pre 1.0) times of julia this was parsed as an Int64? I collect all of your changes for now, test them locally and push them all at once